coruscant/style/theme.js

```diff
@@ -8,6 +8,49 @@ const theme = {
     large: 48
   },
 
+  projectDetails: {
```

Comment:
We should only have styling here that is common for many parts of the application. If this is being used only in the projectDetails page/components, this should be moved there.